java-source/build.gradle

```diff
@@ -0,0 +1,115 @@
+apply plugin: 'net.corda.plugins.quasar-utils'
+apply plugin: 'maven-publish'
+
+sourceSets {
+    main {
+        resources {
+            srcDir "../config/dev"
+        }
+    }
+    test {
+        resources {
+            srcDir "../config/test"
+        }
+    }
+}
+
+
+dependencies {
+    testCompile "junit:junit:$junit_version"
+
+    // Corda integration dependencies
+    cordaCompile "$corda_release_distribution:corda-core:$corda_release_version"
+    cordaCompile "$corda_release_distribution:corda-finance:$corda_release_version"
+    cordaCompile "$corda_release_distribution:corda-jackson:$corda_release_version"
+    cordaCompile "$corda_release_distribution:corda-rpc:$corda_release_version"
+    cordaCompile "$corda_release_distribution:corda-node-api:$corda_release_version"
+    cordaCompile "$corda_release_distribution:corda-webserver-impl:$corda_release_version"
+    cordaRuntime "$corda_release_distribution:corda:$corda_release_version"
+    cordaRuntime "$corda_release_distribution:corda-webserver:$corda_release_version"
+
+    testCompile "$corda_release_distribution:corda-test-utils:$corda_release_version"
+    testCompile "$corda_release_distribution:corda-node-driver:$corda_release_version"
+
+
+    // GraphStream: For visualisation (required by TemplateClientRPC app)
+    compile "org.graphstream:gs-core:1.3"
+    compile("org.graphstream:gs-ui:1.3") {
+        exclude group: "bouncycastle"
+    }
+
+
+    // CorDapp dependencies
+    // Specify your cordapp's dependencies below, including dependent CorDapps
+    cordapp "$corda_release_distribution:corda-finance:$corda_release_version"
+}
+
+tasks.withType(JavaCompile) {
+    options.compilerArgs << "-parameters" // Required for shell commands.
+}
+
+def rpcUsersList = [
+        ['username' : "user1",
+         'password' : "password",
+         'permissions' : ["ALL"]]
+]
+
+task deployNodes(type: net.corda.plugins.Cordform, dependsOn: ['jar']) {
+    delete "./build/nodes"
+    directory "./build/nodes"
+
+    node {
+        name "O=Notary,L=London,C=GB"
+        notary = [validating : false]
+        p2pPort 10006
+        cordapps = ["$corda_release_distribution:corda-finance:$corda_release_version"]
+    }
+    node {
+        name "O=PartyA,L=London,C=GB"
+        p2pPort 10007
+        rpcSettings {
+            address("localhost:10008")
+            adminAddress("localhost:10048")
+        }
+        webPort 10009
+        cordapps = ["$corda_release_distribution:corda-finance:$corda_release_version"]
+        rpcUsers = rpcUsersList
+    }
+    node {
+        name "O=PartyB,L=New York,C=US"
+        p2pPort 10010
+        rpcSettings {
+            address("localhost:10011")
+            adminAddress("localhost:10051")
+        }
+        webPort 10012
+        cordapps = ["$corda_release_distribution:corda-finance:$corda_release_version"]
+        rpcUsers = rpcUsersList 
+    }
+    node {
+        name "O=PartyC,L=Paris,C=FR"
```

Comment:
Why did you change this to Paris? Is doesn't matter which cities we use but they need to be the same